std_msgs/Header header
sensor_msgs/Imu IMU
sensor_msgs/MagneticField MagField